SparkPost/2web2ui
# UX-80 Signals Prototype

#### For a user test on May 20 @ 1pm

### Whats changed
- Reverts SparkPost/2web2ui#1015
- Adds a default component edge instead of a button when no components selected

### To verify
- Verify no page load without UI flag `prototype_signals_details`
- Verify prototype loads with flag (use appteam)
- Verify code splitting

### To do
- [ ] Should I not do this in a revert branch?
